android/walletlib/src/main/java/com/solana/mobilewalletadapter/walletlib/scenario/Scenario.java

```diff
@@ -332,6 +342,7 @@ public interface Callbacks {
         // Request callbacks
         void onAuthorizeRequest(@NonNull AuthorizeRequest request);
         void onReauthorizeRequest(@NonNull ReauthorizeRequest request);
+        void onDeauthorizeRequest(@NonNull DeauthorizeRequest request);
```

Comment:
Maybe we should name the callback onDeauthorizedRequest to reflect the fact that the user has already been deauthorised?

Reply:
👍 will rename